src/main/java/org/terasology/tutorialWorldGeneration/HouseRasterizer.java

```diff
@@ -47,14 +47,15 @@ public void generateChunk(CoreChunk chunk, Region chunkRegion) {
             Vector3i centerHousePosition = new Vector3i(entry.getKey());
             int extent = entry.getValue().getExtent();
             centerHousePosition.add(0, extent, 0);
-            BlockRegion walls = BlockRegion.createFromCenterExtents(centerHousePosition, extent);
-            BlockRegion inside = BlockRegion.createFromCenterExtents(centerHousePosition, extent - 1);
+            BlockRegion walls = new BlockRegion(centerHousePosition);
+            BlockRegion inside = new BlockRegion(walls).expand(extent - 1, extent - 1, extent - 1);
+            walls.expand(extent, extent, extent);
```

Comment:
Is there any specific reason you moved `walls.expand(extent, extent, extent);` last here, and do it in a spearate statement? Do you see any advantage in doing this?

What do you think of keeping it the same as before, creating both regions from the center position and respective extents?
```suggestion
            BlockRegion walls = new BlockRegion(centerHousePosition).expand(extent, extent, extent);
            BlockRegion inside = new BlockRegion(centerHousePosition).expand(extent - 1, extent - 1, extent - 1);
```

Another option might be to create the outer wall region first, and then shrink the region again by the wall thickness.
```suggestion
            BlockRegion walls = new BlockRegion(centerHousePosition).expand(extent, extent, extent);
            BlockRegion inside = new BlockRegion(walls).expand(-1, -1, -1);
```

Reply:
Thanks, these look like good ideas (and I realize I missed some discussion about them above). I went ahead and made the changes.